### ISSUE TITLE
make category result filters apply to torrents as well

### ISSUE THREAD

**Issue by johnwchambers** [theotherp/nzbhydra2#268]
I had a lot of 'forbidden words' setup in the Categories (for Movies/TV/Anime) tab to restrict releases that contained non-English audio and other unwanted formats.  This worked perfectly fine when content was grabbed by sonarr/radarr and sent to nzbget.

However, these 'forbidden words' from the Category tab don't apply to torrent based searches. I had to move all of the 'forbidden words' to the main Search tab Result Filtering section instead to get them to apply.

The side effect of this is that I had different sets of words for Movies & TV shows versus Anime. For Anime I wanted to allow dual audio and etc while for Movies & TV I did not. So in the end, to get the 'forbidden words' to apply globally, I just stuck with unwanted formats instead and have removed all words referencing non-English languages.

Current Setup on the Searching Tab:
![image](https://user-images.githubusercontent.com/752262/50263529-80bd4980-0459-11e9-8778-db499748a0a9.png)

**Comment by johnwchambers** [comment]
Awesome, thanks! I'll move them back to the appropriate section on the Categories tab and let you know how it goes.

**Comment by alexdelifer** [comment]
The trouble now, is that in Jackett, multiple trackers will use the same 10000+ Torznab cat for different things, PTP uses 

Category | Description
-- | --
100001 | Feature Film
100002 | Short Film

while AHD uses

Category | Description
-- | --
100001 | Movies
100002 | TV-Shows

but on TheGeeks it's

100001 | Languages / Linguistics
-- | --



it's not the best example, it's worse in the higher up categories...



This means I'm getting a lot of issues in my categories since many categories overlap, so I can't use any of the 100000+ without getting a bunch in another tracker rejected by hydra search, what would be the best in my opinion would be if the torznab categories to hydra mapping was done on an indexer level, maybe with a little list of all the indexer's capable categories to choose from. This would really help people with lots of trackers in Jackett and Hydra.

Now that this is changed, I had to remove all my 100000+ cats from the categories page, because while it didn't filter them out, it did tag them in Hydra properly, so I could sort by categories in the search list to find what I wanted...

**Comment by theotherp** [comment]
@alexdelifer Sorry, I didn't assume anybody used the custom categories...

The whole category thing is a huge PITA because while there are predefined categories half of the indexers don't care to use them or use them differently.

I've thought about indexer-specific category mapping via GUI but decided against it because it would be terribly hard to design, implement and configure. I already do some mapping internally.

I did a short test with PTP on Jackett and it returned all the movies with category 2000. IPT uses some custom ones but also the predefined ones.

I don't understand why you had to remove the custom categories from the config. All I changed was that they weren't used for category recognition, so while now some categories might not be recognized properly there should be no misidentification of categories.

So, please explain why my change has things made worse for you.

**Comment by theotherp** [comment]
Ooops, I removed categories <99_999 (instead of 9_999)...

**Comment by theotherp** [comment]
See if that helps.

**Comment by alexdelifer** [comment]
This seems to fix most of it for me. Thank you so much!

I had to remove the category 100003 

GFXNews & CGPeers Jackett

Category | Description
-- | --
100003 | Tutorials

from my Other category where I search for E-Learning material since it matches a bunch of stuff from other trackers, they all get categorized as Other in my searches. Maybe if you could allow something like `7000,100003` or `100003,7000` in the categories box, so we can get more granular categories for the trackers that list all the categories per torrent.

Maybe I'm understanding it wrong, but I think this would solve it completely for me.

**Comment by johnwchambers** [comment]
Just wanted to chime in a say all is well for me. Looks like my forbidden words are applying to torrents as well.  Thanks again!

**Comment by theotherp** [comment]
@johnwchambers Great to hear!

@alexdelifer You mean that an item would need to have both numbers in order for this category to be assigned?

**Comment by alexdelifer** [comment]
Exactly, some way to say it needs to be Other/E-Learning (7000,100003) and not (2000,100003) Movie/Foreign.
